scripts/minikube.sh

```diff
@@ -130,17 +130,6 @@ function disable_storage_addons() {
     ${minikube} addons disable storage-provisioner 2>/dev/null || true
 }
 
-# minikube has the Busybox losetup, and that does not work with raw-block PVCs.
```

Comment:
removing this one will cause any issue for the one who is using an older minikube version?